resources/type-system/inference.md

```diff
@@ -474,42 +462,78 @@ replaced with `Null`, and every covariant occurrence of `Ti` replaced with
   from any of the `Bi`:
```

Comment:
I don't understand the "from any of the `Bi`" part here and above.

There are descriptions, which I would understand, don't know which of them are right.
1. "`L` contains any of `Xi`"
2. "any of `Bi` references a type variable from `L`"

Reply:
The latter.  Rephrased to be clearer.